plugin-server/src/cdp/services/hog-executor.service.ts

```diff
@@ -83,12 +85,27 @@ export const formatHogInput = (bytecode: any, globals: HogFunctionInvocationGlob
     if (Array.isArray(bytecode)) {
         return bytecode.map((item) => formatHogInput(item, globals, key))
     } else if (typeof bytecode === 'object' && bytecode !== null) {
-        return Object.fromEntries(
-            Object.entries(bytecode).map(([key2, value]) => [
-                key2,
-                formatHogInput(value, globals, key ? `${key}.${key2}` : key2),
-            ])
-        )
+        let ret: Record<string, any> = {}
+
+        if (bytecode[EXTEND_OBJECT_KEY]) {
```

Comment:
This will have to be modified to support the other comment - it should check for the key and extend the object out but still _additionally_ add the other properties as well